leetcode/views/result.py

```diff
@@ -102,6 +106,51 @@ def make_compile_error_view(self):
         ]
         return urwid.Padding(urwid.ListBox(urwid.SimpleListWalker(list_items)), left=2, right=2)
 
+    def make_runtime_error_view(self):
+        blank = urwid.Divider()
+        status_header = urwid.AttrWrap(urwid.Text('Run Code Status: '), 'body')
+        status = urwid.AttrWrap(urwid.Text('Runtime Error'), 'hometag')
+        columns = urwid.Columns([(17, status_header), (20, status)])
+        column_wrap = urwid.WidgetWrap(columns)
+        result_header = urwid.Text('--- Run Code Result: ---', align='center')
+        your_input_header = urwid.Text('Last input:')
+        your_input = urwid.Text(self.result['last_testcase'])
+        your_answer_header = urwid.Text('Error:')
+        your_answer = urwid.Text(self.result['runtime_error'])
+        expected_answer_header = urwid.Text('')
+        expected_answer = urwid.Text('')
+        list_items = [
+                result_header,
+                blank, column_wrap,
+                blank, your_input_header, your_input,
+                blank, your_answer_header, your_answer,
+                blank, expected_answer_header, expected_answer
+        ]
+        return urwid.Padding(urwid.ListBox(urwid.SimpleListWalker(list_items)), left=2, right=2)
+
+    def make_timeout_view(self):
+        blank = urwid.Divider()
+        status_header = urwid.AttrWrap(urwid.Text('Run Code Status: '), 'body')
+        status = urwid.AttrWrap(urwid.Text('Time Limit Exceeded'), 'hometag')
+        columns = urwid.Columns([(17, status_header), (20, status)])
+        column_wrap = urwid.WidgetWrap(columns)
+        result_header = urwid.Text('--- Run Code Result: ---', align='center')
+        your_input_header = urwid.Text('Last executed input:')
+        your_input = urwid.Text(self.result['last_testcase'])
+        your_answer_header = urwid.Text('')
```

Comment:
"answer" and "expected_answer" related UI element can be removed

Reply:
Fixed. 

Leetcode:
![image](https://user-images.githubusercontent.com/1173734/29901082-e801fa04-8daa-11e7-9570-b3412d6f3de9.png)

Terminal:
![image](https://user-images.githubusercontent.com/1173734/29901092-ed7d2972-8daa-11e7-8d9d-6f8119883ab6.png)